src/routes/e2e/widget/+page.svelte

```diff
@@ -12,30 +12,38 @@
   }
 
   onMount(async () => {
-    configuration.set({
-      clientId: 'WebOAuthClient',
-      redirectUri: `${window.location.origin}/callback`,
-      scope: 'openid profile email me.read',
-      serverConfig: {
-        baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
-        timeout: 5000,
-      },
-      realmPath: 'alpha',
+    configuration().set({
+      config: {
+        clientId: 'WebOAuthClient',
+        redirectUri: `${window.location.origin}/callback`,
+        scope: 'openid profile email me.read',
+        serverConfig: {
+          baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
+          timeout: 5000,
+        },
+        realmPath: 'alpha',
+      }
     });
 
-    // Fetch tokens locally to see if the user has tokens
-    if (await user.tokens()) {
-      fetchedTokens = true;
-      userInfo = await user.info(true);
-    } else {
-      // Set fetched tokens to true so the links render to login
-      fetchedTokens = true;
-    }
+    // Using observable method:
+    const { get, subscribe } = user.info();
+    get();
+    subscribe((event) => {
+      console.log(event);
+      loading = event.loading;
+      userInfo = event.response;
+    });
+
+    // Using Promise method
+    // const { get } = user.info();
+    // const event = await get();
+    // console.log(event);
+    // userInfo = event.response;
```

Comment:
Are we keeping these comments in? Guessing so as it's showing an alternative way to do something but thought I'd check.

Reply:
I'm a bit more liberal with what is left behind in these e2e test apps/pages. I'm just illustrating that either are acceptable. In the near future, it might be best to have a switch that gives us a control the flip between the two variations.